### DISCUSSION TITLE
Smart Button ?

### DISCUSSION THREAD

**Discussion by pulse-mind** [spookylukey/django-paypal#277] [Q&A] [answered]
Hello,

Thank you for the job done on django-paypal.
I'd like to know if it is possible to have Smart Button: https://developer.paypal.com/docs/checkout/standard/integrate/ using this plugin? 

Best regards
Fred

**Comment by newearthmartin** [comment, marked as answer]
Django-paypal does not cover the JavaScript smart button part but it is still very useful for receiving and processing payment notifications such as IPN. You would need to write the JavaScript part yourself. That is really not much more than the codes in the page that you linked.

**Reply by pulse-mind** [reply]
Thanks for your answer :).
